### DISCUSSION TITLE
Math mode expressions overlap in output file

### DISCUSSION THREAD

**Discussion by TehpphoENIX** [typst/typst#2811] [General]
### Description

When I was writing this typst markup:
```typst
- $a_i = limits(sum)_(j=1)^p w_(i,j)y_j$ -- стимул нейрона $i$, де $w_(i,j)$ -- елемент матриці $W$
- $y_i = cases(1 "if" a_i > 0, y_i "if" a_i = 0, -1 "if" a_i < 0)$
```
It overlapped in output
![image](https://github.com/typst/typst/assets/91074010/b1923d07-bc93-4a71-a927-1e772a888714)

### Operating system

Linux

### Typst version

- [X] I am using the latest version of Typst

**Comment by TehpphoENIX** [comment]
They overlap at approximately 1 em.

**Reply by Enivex** [reply]
They overlap because you're using inline math, which has purposely smaller bounding boxes to avoid shifting line spacing.

**Reply by TehpphoENIX** [reply]
But how you use non-inline math in Typst?

**Reply by Enivex** [reply]
> But how you use non-inline math in Typst?

You add whitespace at each end

$a$ is inline,

while

$ a $

or

$
    a
$

is display.
